WordCount/WordCountAPI/JsonModels/JsonValidator.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Text.Json;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Schema;
+
+namespace WordCount.JsonModels
+{
+    public sealed class JsonValidator<T> where T : class 
```

Comment:
Really nice with sealed classes. Maybe we should consider making a IPostRequestValidator interface for quick switching?

Reply:
This looks really good! Really nice with a schema string as input in constructor!